recipes/slow-buffer-to-buffer-alloc-unsafe-slow/src/workflow.ts

```diff
@@ -0,0 +1,296 @@
+	const importStatements = getNodeImportStatements(root, "buffer");
+
+	for (const statement of importStatements) {
+		// Try to find named_imports directly in the statement
+		const namedImports = statement.find({ rule: { kind: "named_imports" } });
+		if (namedImports) {
+			const originalText = namedImports.text();
+			if (originalText.includes("SlowBuffer")) {
+				let newText = transformDestructuringPattern(originalText);
+				if (newText !== originalText) {
+					edits.push(namedImports.replace(newText));
+					hasChanges = true;
+				}
+			}
+		}
+	}
+
+	return hasChanges;
+}
+
+/**
+ * Transform destructuring patterns by replacing SlowBuffer with Buffer
+ * Handles cases like { SlowBuffer }, { Buffer, SlowBuffer }, { SlowBuffer, Buffer }, etc.
+ * Also handles aliased patterns like { SlowBuffer: SB }, { SlowBuffer: SB, Buffer }
+ * For ESM imports: { SlowBuffer as SB } -> { Buffer as SB }
+ * Preserves original formatting as much as possible
+ */
+function transformDestructuringPattern(originalText: string): string {
+	// For simple case { SlowBuffer }, replace with { Buffer } preserving whitespace
+	if (
+		originalText.includes("SlowBuffer") &&
+		!originalText.includes(",") &&
+		!originalText.includes(":") &&
+		!originalText.includes(" as ")
+	) {
+		return originalText.replace(/SlowBuffer/g, "Buffer");
+	}
+
+	let newText = originalText;
+
+	// Handle aliased patterns (both CommonJS : and ESM as syntax)
+	// { SlowBuffer: SB } -> { Buffer: SB }
+	newText = newText.replace(/SlowBuffer(\s*:\s*\w+)/g, "Buffer$1");
+	// { SlowBuffer as SB } -> { Buffer as SB }
+	newText = newText.replace(/SlowBuffer(\s+as\s+\w+)/g, "Buffer$1");
+
+	// If Buffer is already present in this specific pattern, just remove SlowBuffer
+	if (originalText.includes("Buffer") && originalText.includes("SlowBuffer")) {
+		// Remove non-aliased SlowBuffer references very carefully to preserve spacing
+		newText = newText
+			.replace(/,\s*SlowBuffer(?!\s*[:as])/g, "") // Remove SlowBuffer with leading comma
+			.replace(/SlowBuffer\s*,\s*/g, "") // Remove SlowBuffer with trailing comma and space
+			.replace(/SlowBuffer(?!\s*[:as])/g, ""); // Remove standalone SlowBuffer (not followed by : or as)
+
+		// Clean up any double spaces after opening brace
+		newText = newText.replace(/{\s{2,}/g, "{ ");
+		// Clean up any double commas but preserve spacing
+		newText = newText.replace(/,\s*,/g, ",");
+	}
+	// If Buffer is not present, replace first SlowBuffer with Buffer, remove others
+	else if (originalText.includes("SlowBuffer")) {
+		// Replace the first non-aliased SlowBuffer with Buffer
+		newText = newText.replace(/SlowBuffer(?!\s*[:as])/, "Buffer");
+		// Remove any remaining non-aliased SlowBuffer references
+		newText = newText
+			.replace(/,\s*SlowBuffer\s*(?![\s:as])/g, "")
+			.replace(/SlowBuffer\s*,\s*/g, "")
+			.replace(/SlowBuffer\s*(?![\s:as])/g, "");
+		// Clean up commas
+		newText = newText.replace(/,\s*,/g, ",");
+	}
+
+	return newText;
```

Comment:
I think it can be handled with `updateBinding` utility function  https://github.com/nodejs/userland-migrations/pull/182 , but it hasn’t been merged yet.

@AugustinMauroy Could we check with someone to follow up on the #182 merge?